### PR TITLE
docs: add special config steps to make CephFS CSI v1.1.0

### PR DESCRIPTION
**Pull request by ajarr**
... work with Ceph v14.2.2 cluster that is not deployed by rook.

Fixes: #486
Signed-off-by: Ramana Raja <rraja@redhat.com>

### CODE REVIEW

File: docs/deploy-cephfs.md [hunk 1]
```diff
@@ -26,6 +26,19 @@ make image-cephcsi
 
 ## Configuration
 
+**NOTE:** To make CephFS CSI driver version >= 1.1.0 work with Ceph v14.2.2
+cluster (not deployed by rook), you need to add settings in the `mgr` section
```
Comment (ShyamsundarR) [inline docs/deploy-cephfs.md]: (nit) reword slightly?

"To make CephFS CSI driver version >= 1.1.0 work with Ceph v14.2.2 cluster (not deployed by rook), you need to add the following settings, in the `mgr` section of the ceph.conf used by the Ceph manager daemon, and restart the Ceph manager deamon.

```
[mgr]
client mount uid = 0
client mount gid = 0
```

This is due to an [issue](http://tracker.ceph.com/issues/40927) in Ceph v14.2.2 that should be resolved in v14.2.3."

Comment (ajarr) [reply docs/deploy-cephfs.md]: Agreed. Reads better. Thanks! Please merge this if it looks OK to you.

Comment (ShyamsundarR) [reply docs/deploy-cephfs.md]: Approved.

Unfortunately merging is auto-bot driven based on 2 approvals (although there seems to be a time out based call to merge manually at times as well). So, I would wait for one more approval before we merge this, to stick to repository working procedures.

Tagging @Madhu-1 @poornimag @humblec to provide the second approval to move this forward.

File: docs/deploy-cephfs.md [hunk 2]
```diff
@@ -26,6 +26,20 @@ make image-cephcsi
 
 ## Configuration
 
+**NOTE:** To make CephFS CSI driver version >= 1.1.0 work with Ceph v14.2.2
+cluster (not deployed by rook), you need to add the following settings in the
+`mgr` section of the ceph.conf used by the Ceph manager daemon, and restart the
+Ceph manager daemon.
+
+```
```
Comment (Madhu-1) [inline docs/deploy-cephfs.md]: add bash to the markdown

Comment (ShyamsundarR) [reply docs/deploy-cephfs.md]: This is a section in the ceph.conf configuration file, and not a bash command/shell execution, is it still required?

Comment (Madhu-1) [reply docs/deploy-cephfs.md]: there is no way to highlight the general config file content in markdown. so with these changes, the configuration won't be highlighted

**Pull request merged**